### ISSUE TITLE
Processing was stopped out , Upsampling is deprecated

### ISSUE THREAD

**Issue by firerflystar** [braindotai/Watermark-Removal-Pytorch#10]
Starting training...

Completed:   0%|                                                              | 0/9 [00:00<?, ?it/s]C:\Program Files\Python36\lib\site-packages\torch\nn\modules\upsampling.py:122: UserWarning: nn.Upsampling is deprecated. Use nn.functional.interpolate instead.
  warnings.warn("nn.Upsampling is deprecated. Use nn.functional.interpolate instead.")
Completed: 100%|█████████████████████████████████████████| 9/9 [00:12<00:00,  1.43s/it, Loss=0.0263]

Saving final output image to: "E:\Watermark-Removal-Pytorch\demo\frames\00000-output.jpg"
